.circleci/config.yml

```diff
@@ -189,6 +189,7 @@ jobs:
       - image: circleci/python:3.7-stretch-node-browsers
         environment:
           PERCY_PARALLEL_TOTAL: -1
+          PUPPETEER_SKIP_CHROMIUM_DOWNLOAD: True
```

Comment:
What's this about?

Reply:
the npx needs implicitly the installation of puppeteer, which will then download the big guy `chromium`, this option is to skip the download.